.github/workflows/build-example-web.yml

```
@@ -1,43 +1,162 @@
+env:
+  COMMIT_SHORT_SHA: ${{ github.sha }}
+
 jobs:
-  deploy:
+  build_expo:
     runs-on: ubuntu-latest
-    env:
-      TEST_ENDPOINT: hardware-example.onekeytest.com
-      PROJECT_PATH: packages/connect-examples/expo-example
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)